test/controllers/projects/workflow_executions_controller_test.rb

```diff
@@ -8,173 +8,179 @@ class WorfklowExecutionsControllerTest < ActionDispatch::IntegrationTest
+                                                         format: :turbo_stream)
       end
       assert_response :unprocessable_entity
     end
 
     test 'should delete a canceled workflow' do
-      workflow_execution = workflow_executions(:irida_next_example_canceled)
+      workflow_execution = workflow_executions(:automated_example_canceled)
       assert workflow_execution.canceled?
       assert_difference -> { WorkflowExecution.count } => -1,
                         -> { SamplesWorkflowExecution.count } => -1 do
-        delete workflow_execution_path(workflow_execution, format: :turbo_stream)
+        delete namespace_project_workflow_execution_path(@namespace, @project, workflow_execution,
+                                                         format: :turbo_stream)
       end
       assert_response :success
     end
 
     test 'should not delete a running workflow' do
-      workflow_execution = workflow_executions(:irida_next_example_running)
+      workflow_execution = workflow_executions(:automated_example_running)
       assert workflow_execution.running?
       assert_difference -> { WorkflowExecution.count } => 0,
                         -> { SamplesWorkflowExecution.count } => 0 do
-        delete workflow_execution_path(workflow_execution, format: :turbo_stream)
+        delete namespace_project_workflow_execution_path(@namespace, @project, workflow_execution,
+                                                         format: :turbo_stream)
       end
       assert_response :unprocessable_entity
     end
 
     test 'should cancel a running workflow' do
-      workflow_execution = workflow_executions(:irida_next_example_running)
+      workflow_execution = workflow_executions(:automated_example_running)
       assert workflow_execution.running?
 
-      put cancel_workflow_execution_path(workflow_execution, format: :turbo_stream)
+      put cancel_namespace_project_workflow_execution_path(@namespace, @project, workflow_execution,
+                                                           format: :turbo_stream)
       assert_response :success
       # A running workflow goes to the canceling state as ga4gh must be sent a cancel request
       assert_equal 'canceling', workflow_execution.reload.state
     end
 
     test 'should not delete a new workflow' do
-      workflow_execution = workflow_executions(:irida_next_example_new)
+      workflow_execution = workflow_executions(:automated_example_new)
       assert workflow_execution.initial?
       assert_difference -> { WorkflowExecution.count } => 0,
                         -> { SamplesWorkflowExecution.count } => 0 do
-        delete workflow_execution_path(workflow_execution, format: :turbo_stream)
+        delete namespace_project_workflow_execution_path(@namespace, @project, workflow_execution,
+                                                         format: :turbo_stream)
       end
       assert_response :unprocessable_entity
     end
```

Comment:
**nitpick (blocking):** Could you add in a test for deleting with param `redirect: true` and ensure redirecting to the correct location.

Reply:
Testing added in 3db41ad0245de038956cfba245d7855ce5e5ba9c